cypress/integration/examples/codeHighlighting.ts

```diff
@@ -0,0 +1,35 @@
+describe('code highlighting', () => {
```

Comment:
The file name should use the hyphenated convention used elsewhere, so `examples/code-highlighting.ts`

Reply:
Resolved in some other examples too that I noticed have `fileName` different from that in `site/examples` folder 🤗